.sequelizerc

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+const path = require('path')
+
+module.exports = exports = {
+  'config':          path.resolve('database.json'),
+  'migrations-path': path.resolve('migrations'),
+  'models-path': path.resolve('server', 'models')
```

Comment:
neen. `src/models` is typescript en daar kan sequelize-cli niet mee overweg. ik zal eens kijken of ik de models.path volledig kan laten wegvallen